corehq/apps/app_manager/suite_xml/case_tile_templates/tdh.txt

```diff
@@ -58,12 +58,17 @@
       </header>
       <template>
         <text>
-          &lt;b&gt;{header[prefix]} &lt;/b&gt; <xpath function="{header[xpath_function]}"/>
+          &lt;b&gt;{header[prefix]} &lt;/b&gt;
+          <xpath function="{header[xpath_function]}">
+            {header[variables]}
```

Comment:
This looks like a real change (not just whitespace). Was that intended? I wouldn't expect this new feature to change the output for any existing apps or tests